### ISSUE TITLE
Add icon on user's Publications List to indicate that a publication has an “Activity Insight postprint status” of “in progress”

### ISSUE THREAD

**Issue by anaelizabethenriquez** [psu-libraries/researcher-metadata#600]
Since we have this data in RMD and are now suppressing emails based on it, we'd like to reflect it on the [Manage Profile Publications page](https://metadata.libraries.psu.edu/profile/publications/edit) with an icon as well.

The icon should be labeled with the following text: "A file for the publication was uploaded in Activity Insight and is being processed for deposit in ScholarSphere."

**Comment by anaelizabethenriquez** [comment]
Related to #1005

**Comment by jlandiseigsti** [comment]
What would you like the alt-text to be for the icon?

**Comment by anaelizabethenriquez** [comment]
@jlandiseigsti how about something like "Icon of computer drive with arrow", adjusted to be similar to the alt text for the other icons on that page? I tried to view their alt text but did not succeed.

**Comment by jlandiseigsti** [comment]
It looks like instead of a standard alt-text, we have screen-reader only text that translates the icon into words. For example, for the 'In press' icon, it says "Publication is in press and will not be subject to the open access policy until published". This is very close to what it says at the key at the top...do you want me to just use the key wording ('A file for the publication was uploaded in Activity Insight and is being processed for deposit in ScholarSphere.')?

**Comment by anaelizabethenriquez** [comment]
Sounds good, @jlandiseigsti. Thanks!

**Comment by jlandiseigsti** [comment]
I'm guessing we don't want a link to the Explanation of Open Access the way we do for unknown status, but I wanted to confirm that.

**Comment by anaelizabethenriquez** [comment]
That sounds right to me, because this icon indicates that the researcher does not need to take action at this time, so they don't need that info immediately. But I'm not certain where this is that you're asking about -- do we have a link in the screen-reader only text?

**Comment by jlandiseigsti** [comment]
No, the title of the article itself is the link for articles with unknown status.

**Comment by anaelizabethenriquez** [comment]
Ah! You're correct. For articles that get this icon, we should also remove that link.
